.eslintrc

```diff
@@ -0,0 +1,85 @@
+{
+  "parser": "babel-eslint",
```

Comment:
Thank you for walking me through this code. Its great to see the binary tree in action!